web/src/auth/SignUpPage.tsx

```diff
@@ -111,6 +111,12 @@ export class SignUpForm extends React.Component<SignUpFormProps, SignUpFormState
                         )}
                     </button>
                 </div>
+                {window.context.sourcegraphDotComMode && (
+                    <p>
+                        Create a public account to browse and discuss open-source code and manage Sourcegraph
```

Comment:
```suggestion
                        Create a public account to search/navigate open-source code and manage Sourcegraph
```

Remove "discuss" because code discussions is experimental and not enabled by default, and use "search/navigate" to be consistent with the wording used on our homepage